app/assets/stylesheets/application.css.scss

```diff
@@ -53,6 +53,26 @@ body {
   background-position: center center;
 }
 
+.application-body-blacksmith{
+  background-image: url('blacksmith.jpg');
+  background-size:     cover;
```

Comment:
Properties should be ordered background-image, background-position, background-repeat, background-size<br>Colon after property should be followed by one space